### PR TITLE
Enhancing and standardizing Databricks and Delta Lake documentation

### LINKED ISSUE #583: Enhance Databricks connector documentation
Page should have:
- [x] Connector description
- [x] Configuration section
- [x] Secrets section / Authentication section
- [x] Examples section
- [x] Parameters table
- [ ] File format support (if applicable)

### LINKED ISSUE #585: Enhance Delta Lake connector documentation
Page should have:
- [ ] Connector description
- [ ] Configuration section
- [ ] Secrets section / Authentication section
- [ ] Examples section
- [ ] Parameters table
- [ ] File format support (if applicable)

### PR DESCRIPTION
**Pull request by eadgbear**
## 🗣 Description

Standardizing connector documentation as outlined in #580 

## 🔨 Related Issues

- Closes #583 
- Closes #585

### CODE REVIEW

**Review by sgrebnov — DISMISSED** [review]

File: spiceaidocs/docs/components/data-connectors/databricks.md [hunk 1]
```diff
@@ -34,109 +30,146 @@ datasets:
...
 
-
 ## Configuration
 
-Use the [secret replacement syntax](../secret-stores/index.md) to reference a secret, e.g. `${secrets:my_token}`.
+### `from`
 
-<Tabs>
-  <TabItem value="spark_connect" label="Spark Connect" default>
+The `from` field for the Databricks connector takes the form `databricks:schema.database.table` where `schema.database.table` is the fully-qualified path to the table to read from.
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/databricks.md]: It should be `catalog.schema.table` in the Databricks vocabulary, not `schema.database.table`

File: spiceaidocs/docs/components/data-connectors/databricks.md [hunk 2]
```diff
@@ -34,109 +30,146 @@ datasets:
...
+    mode: delta_lake
+    databricks_endpoint: dbc-a1b2345c-d6e7.cloud.databricks.com
+    databricks_token: ${secrets:my_token}
+    databricks_google_service_account_path: /path/to/service-account.json
+```
+
+## Using secrets
 
-#### Google Storage (GCS)
+There are currently three supported [secret stores](/components/secret-stores/index.md):
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/databricks.md]: Aren't all of the secret stores supported?

File: spiceaidocs/docs/components/data-connectors/delta-lake.md [hunk 3]
```diff
@@ -5,113 +5,142 @@ description: 'Delta Lake Data Connector Documentation'
...
+
+```yaml
+    params:
+      delta_lake_google_service_account_path: /path/to/service-account.json
+```
+
+## Using secrets
 
-#### Google Storage (GCS)
+There are currently three supported [secret stores](/components/secret-stores/index.md):
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/delta-lake.md]: Aren't all of the secret stores supported?

File: spiceaidocs/docs/components/data-connectors/databricks.md (hunk 2 above)
Comment (eadgbear) [reply spiceaidocs/docs/components/data-connectors/databricks.md]: Fixed

File: spiceaidocs/docs/components/data-connectors/delta-lake.md (hunk 3 above)
Comment (eadgbear) [reply spiceaidocs/docs/components/data-connectors/delta-lake.md]: Fixed

File: spiceaidocs/docs/components/data-connectors/databricks.md (hunk 1 above)
Comment (eadgbear) [reply spiceaidocs/docs/components/data-connectors/databricks.md]: Fixed

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**